contracts/SlashIndicator.sol

```diff
@@ -284,17 +265,7 @@ contract SlashIndicator is ISlashIndicator, System, IParamSubscriber, IApplicati
             }
         }
 
-        if (IStakeHub(STAKE_HUB_ADDR).voteToOperator(_evidence.voteAddr) != address(0)) {
-            IStakeHub(STAKE_HUB_ADDR).maliciousVoteSlash(_evidence.voteAddr);
-        } else {
-            // send slash msg to bc if the validator not migrated
-            ICrossChain(CROSS_CHAIN_CONTRACT_ADDR).sendSynPackage(
-                SLASH_CHANNELID, encodeVoteSlashPackage(_evidence.voteAddr), 0
-            );
-
-            bytes32 voteAddrSlice = BytesLib.toBytes32(_evidence.voteAddr, 0);
-            emit maliciousVoteSlashed(voteAddrSlice);
-        }
+        IStakeHub(STAKE_HUB_ADDR).maliciousVoteSlash(_evidence.voteAddr);
```

Comment:
We'd better do this to prevent reentrant:
```
        IStakeHub(STAKE_HUB_ADDR).maliciousVoteSlash(_evidence.voteAddr);

        // reward sender and felony validator if validator found
        (address[] memory vals, bytes[] memory voteAddrs) =
            IBSCValidatorSet(VALIDATOR_CONTRACT_ADDR).getLivingValidators();
        for (uint256 i; i < voteAddrs.length; ++i) {
            if (BytesLib.equal(voteAddrs[i], _evidence.voteAddr)) {
                uint256 amount = (address(SYSTEM_REWARD_ADDR).balance * felonySlashRewardRatio) / 100;
                IBSCValidatorSet(VALIDATOR_CONTRACT_ADDR).felony(vals[i]);
                ISystemReward(SYSTEM_REWARD_ADDR).claimRewards(msg.sender, amount);
                break;
            }
        }
```